base1: 'AAAAAAAAAAAAAAAA'

base2: 'BBBBBBBBBBBBBBBB'
